lib/utils/sdk.ts

```diff
@@ -27,10 +32,6 @@ export async function getApplication(
 	nameOrSlugOrId: string | number,
 	options?: PineOptions<Application>,
 ): Promise<Application> {
-	// TODO: Consider whether it would be useful to generally include interactive selection of application here,
-	//       when nameOrSlugOrId not provided.
-	//       e.g. nameOrSlugOrId || (await (await import('../../utils/patterns')).selectApplication()),
-	//       See commands/device/init.ts ~ln100 for example
```

Comment:
I removed this comment (It may have been me that added it), as I think this is not the right layer for UI converns.  This module is just for working with the SDK, and the methods here might be good candidates for incorporation into the SDK.